playground/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "private": true,
+  "name": "lwc-playground",
+  "version": "2.20.0",
+  "description": "Playground project to experiment with LWC.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/salesforce/lwc.git",
+    "directory": "playground"
+  },
```

Comment:
AIUI `repository` is only necessary if we're publishing to npm. So I think this can be removed.